### ISSUE TITLE
bazel test --runs_per_test=1000 //staging/src/k8s.io/client-go/tools/remotecommand/...  failed

### ISSUE THREAD

**Issue by pandaamanda** [kubernetes/kubernetes#97900]
**Which jobs are failing**:

bazel test //staging/src/k8s.io/client-go/tools/remotecommand/...
When --runs_per_test value is big enough, the command will fail.

```
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 167 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_167_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 167 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 167 of 1000):
E0111 11:36:51.959802      22 v2.go:151] return err
E0111 11:36:51.959952      22 v2.go:105] client crashed after 1 second
--- FAIL: TestSPDYExecutorStream (32.29s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 170 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_170_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 170 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 170 of 1000):
E0111 11:36:52.210420      23 v2.go:151] return err
E0111 11:36:52.211765      23 v2.go:105] client crashed after 1 second
--- FAIL: TestSPDYExecutorStream (32.05s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 948 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_948_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 948 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 948 of 1000):
E0111 11:36:57.077775      23 v2.go:151] return err
E0111 11:36:57.078183      23 v2.go:105] client crashed after 1 second
--- FAIL: TestSPDYExecutorStream (31.97s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 794 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_794_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 794 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 794 of 1000):
E0111 11:37:11.445719      20 v2.go:151] return err
E0111 11:37:11.446438      20 v2.go:105] client crashed after 1 second
--- FAIL: TestSPDYExecutorStream (32.27s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 700 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_700_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 700 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 700 of 1000):
E0111 11:37:20.779494      22 v2.go:151] return err
E0111 11:37:20.779494      22 v2.go:105] client crashed after 1 second
--- FAIL: TestSPDYExecutorStream (32.32s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 635 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_635_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 635 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 635 of 1000):
E0111 11:37:25.760589      22 v2.go:105] client crashed after 1 second
E0111 11:37:55.774327      22 v2.go:105] client crashed after 1 second
E0111 11:37:55.774473      22 v2.go:168] return err
--- FAIL: TestSPDYExecutorStream (32.09s)
    remotecommand_test.go:145: stdoutBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 557 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_557_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 557 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 557 of 1000):
E0111 11:37:34.814165      21 v2.go:105] client crashed after 1 second
E0111 11:37:34.824313      21 v2.go:151] return err
--- FAIL: TestSPDYExecutorStream (32.33s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 416 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_416_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 416 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 416 of 1000):
E0111 11:37:47.333758      19 v2.go:151] return err
E0111 11:37:47.340509      19 v2.go:105] client crashed after 1 second
--- FAIL: TestSPDYExecutorStream (32.15s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 276 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_276_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 276 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 276 of 1000):
E0111 11:37:58.731507      23 v2.go:105] client crashed after 1 second
E0111 11:37:58.731646      23 v2.go:151] return err
--- FAIL: TestSPDYExecutorStream (32.17s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 269 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_269_of_1000/test.log)
INFO: From Testing //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 269 of 1000):
==================== Test output for //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 269 of 1000):
E0111 11:37:58.820695      21 v2.go:105] client crashed after 1 second
E0111 11:37:58.820830      21 v2.go:151] return err
--- FAIL: TestSPDYExecutorStream (32.41s)
    remotecommand_test.go:145: stderrBlockTest: expected [], got [execute timeout]
FAIL
================================================================================
FAIL: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (run 57 of 1000) (see /root/.cache/bazel/_bazel_root/d9df0a55ef8011e4541f177f4cd64056/execroot/io_k8s_kubernetes/bazel-out/k8-fastbuild/testlogs/staging/src/k8s.io/client-go/tools/remotecommand/go_default_test/run_57_of_1000/test.log)

FAILED: //staging/src/k8s.io/client-go/tools/remotecommand:go_default_test (Summary)
```

**Which test(s) are failing**:

**Since when has it been failing**:

**Testgrid link**:

**Reason for failure**:

**Anything else we need to know**:

**Comment by pandaamanda** [comment]
Maybe duplicated with #97331, i will track it.

**Comment by pandaamanda** [comment]
Fixed in #97909, close it.
/close
